tesseract_common/include/tesseract_common/types.h

```diff
@@ -160,6 +160,29 @@ struct ContactManagersPluginInfo
   // Yaml Config key
   static inline const std::string CONFIG_KEY{ "contact_manager_plugins" };
 };
+
+/** @brief The CalibrationInfo struct */
+struct CalibrationInfo
+{
+  EIGEN_MAKE_ALIGNED_OPERATOR_NEW
+
+  CalibrationInfo() = default;
+
+  /** @brief The joint origin calibration information */
+  tesseract_common::TransformMap joints;
```

Comment:
Can we get more documentation on what this does? Does it replace the existing map or get applied on top of it?

Reply:
For every entry it applies a `ChangeJointOriginCommand`, so it is replacing the origin of the joint with what was provided.